vrc-get-gui/locales/ja.json5

```
@@ -29,13 +29,13 @@
     "projects:error:load error": "プロジェクトリストの読込中にエラーが発生しました: {{msg}}",
     "projects:toast:unity exits with non-zero": "Unityが0以外の終了コードで終了しました。",
     "projects:toast:project migrated": "プロジェクト移行に成功しました。",
-    "projects:tooltip:no directory": "プロジェクトディレクトリが存在しません。",
+    "projects:tooltip:no directory": "元の場所でプロジェクトが見つかりませんでした。",
```

元の場所...?移動してない場合のほうが多いから移動を前提にした表現は違和感が強い

---

登録時の場所、という意味合いが感じられると思ったので、駄目でも無いかなぁと思い…
とはいえ、削除したケースで気になるといわれるとそんな気もします。

その場所、この場所という表現は、"その"や"この"が指す場所が日本語では不明瞭に感じたので避けています。(一応画面上には保存場所のパスが出てはいますが…)
そのため、"元の"や"元あった"などを案として思い浮かべて一旦今の形になりました。

(改行して2行にして、削除と移動の両方を想定した説明文を入れるとかも検討したんですが、ボタン上のツールチップはbr非対応だったのでわざわざ対応させなくて良いかなと思ったなどもあります)

---

`プロジェクトが見つかりませんでした。`ではだめかな

---

それでも良いかなと思います
```suggestion
    "projects:tooltip:no directory": "プロジェクトが見つかりませんでした。",
```
(個人的にはもうちょっと情報量を与えたかった感じですが…ちょっと今思いついたことがあるのでそっちで代えられるかもしれません)

---

> (個人的にはもうちょっと情報量を与えたかった感じですが…ちょっと今思いついたことがあるのでそっちで代えられるかもしれません)

待ったほうがいい？

---

待たなくて大丈夫です。feature requestなので